nav2_controller/plugins/simple_goal_checker.cpp

```diff
@@ -105,6 +119,30 @@ bool SimpleGoalChecker::isGoalReached(
   return fabs(dyaw) < yaw_goal_tolerance_;
 }
 
+void
+SimpleGoalChecker::on_parameter_event_callback(
+  const rcl_interfaces::msg::ParameterEvent::SharedPtr event)
+{
+  for (auto & changed_parameter : event->changed_parameters) {
+    const auto & type = changed_parameter.value.type;
+    const auto & name = changed_parameter.name;
+    const auto & value = changed_parameter.value;
+
+    if (type == ParameterType::PARAMETER_DOUBLE) {
+      if (name == plugin_name_ + ".xy_goal_tolerance") {
```

Comment:
Wouldn't this be more than just name + parameter? Wouldn't it also be in the controller_server namespace at minimum? This doesn't have its own node. Did you print out the `name` to make sure this full path is valid?

Reply:
Yes, same logic as for NavFn I believe.
If I do a print in the code:
```
[controller_server-7] [WARN] [SimpleGoalChecker]: name: goal_checker.xy_goal_tolerance
[controller_server-7] [WARN] [SimpleGoalChecker]: plugin_name_: goal_checker
```

Reply:
Ok, works for me then, just verifying